1.11/cli/command-reference/dcos-license/dcos-license-status/index.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: layout.pug
+navigationTitle:  dcos license status
+title: dcos license status
+menuWeight: 4
+excerpt:
+
+enterprise: true
+---
+
+# Description
+Get the cluster license status.
+
+# Usage
+
+```bash
+dcos license status  [--terms] [--breaches]
+```
+
+# Options
+
+| Name, shorthand | Default | Description |
+|---------|-------------|-------------|
+| `--terms`   |             |  Specify the file path to store the license. |
```

Comment:
`Specify the file path to store the license.` is wrong. 
--terms in to display only the terms of contract
--breaches displays only the number of breaches

Reply:
I am not sure of the wording. Please be explicit. My entry should be double checked.

Reply:
I made this change.